src/gpuarray_blas_cuda_cublas.c

```diff
@@ -520,6 +520,25 @@ static int hgemmBatch(cb_order order, cb_transpose transA, cb_transpose transB,
                       gpudata **B, size_t *offB, size_t ldb,
                       float beta, gpudata **C, size_t *offC, size_t ldc,
                       size_t batchCount) {
+  (void)order;
+  (void)transA;
+  (void)transB;
+  (void)M;
+  (void)N;
+  (void)K;
+  (void)alpha;
+  (void)A;
+  (void)offA;
+  (void)lda;
+  (void)B;
+  (void)offB;
+  (void)ldb;
+  (void)beta;
+  (void)C;
+  (void)offC;
+  (void)ldc;
+  (void)batchCount;
+
```

Comment:
@abergeron They're gone and the tests pass. When `CMAKE_C_FLAGS` is `-Wall -Wextra -Wno-unused-parameter` the only warnings are out of `gpuarray_reduction.c`, which the other PR handles.